_commands/basics/rm.md

```diff
@@ -0,0 +1,55 @@
+---
+---
+
+rm
+--
+
+`rm` is used to delete one or multiple files from your current directory. 
```

Comment:
Might be worth nothing that this does __not__ move to trash, but will delete the file permanently/immediately. Be careful!